OpenRA.Mods.RA/Graphics/TeslaZapRenderable.cs

```diff
@@ -35,13 +35,15 @@ struct TeslaZapRenderable : IRenderable, IFinalizedRenderable
 		readonly WVec length;
 		readonly string image;
 		readonly string palette;
+		readonly string dimSequence;
+		readonly string brightSequence;
 		readonly int brightZaps, dimZaps;
 
 		WPos cachedPos;
 		WVec cachedLength;
 		IEnumerable<IFinalizedRenderable> cache;
 
-		public TeslaZapRenderable(WPos pos, int zOffset, WVec length, string image, int brightZaps, int dimZaps, string palette)
+		public TeslaZapRenderable(WPos pos, int zOffset, WVec length, string image, int brightZaps, int dimZaps, string palette, string dim, string bright)
```

Comment:
The order of the arguments here is inconsistent: it specifies brightZaps followed by dimZaps, but then dim followed by bright.

To improve readability please change this to `public TeslaZapRenderable(WPos pos, int zOffset, WVec length, string image, string brightZapsSequence, int brightZapsCount, string dimZapsSequence, int dimZapsCount, string palette)`